src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml.cs

```diff
@@ -148,6 +161,24 @@ private void ItemsList_SelectionChanged(object sender, SelectionChangedEventArgs
         }
     }
 
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("CodeQuality", "IDE0051:Remove unused private members", Justification = "VS is too aggressive at pruning methods bound in XAML")]
```

Comment:
(Forgot to ask, I guess maybe we've suppressed this warning in our repo (IDE0051), but do you have an issue filed for this somewhere in VS or the WinUI repo? This seems like something that should get fixed there.)